releaseNotes/MicrosoftExtensionsConfigurationAzureAppConfiguration.md

```diff
@@ -1,6 +1,13 @@
 ## Microsoft.Extensions.Configuration.AzureAppConfiguration
 ### [Package (NuGet)](https://www.nuget.org/packages/Microsoft.Extensions.Configuration.AzureAppConfiguration)
 
+### 7.0.0-preview.2 - August 24, 2023
+
```

Comment:
Should we add a note clarifying that this version contains changes released in 6.1.0?

Reply:
I considered it, I think it's implied by the versions but we could say "Includes all changes made in `6.1.0`." for anyone confused by the order of releases.

Reply:
That would be sufficient. I just thought it was odd to see empty release notes for this version.